README.md

```diff
@@ -1,111 +1,105 @@
 
-### 3. Run docker container
+This enables the most popular open-source models, including Transformer-like, Mixture-of-Expert, Embedding, Multi-modal LLMs to run seamlessly on Ascend NPU.
 
-> [!NOTE]
-> Modify `--device /dev/davinci0` according to your device.
+## Prerequisites
+### Support Devices
+- Atlas A2 Training series (Atlas 800T A2, Atlas 900 A2 PoD, Atlas 200T A2 Box16, Atlas 300T A2)
+- Atlas 800I A2 Inference series (Atlas 800I A2)
 
-```bash
-docker run -dit -v /usr/local/dcmi:/usr/local/dcmi -v /usr/local/bin/npu-smi:/usr/local/bin/npu-smi -v /usr/local/Ascend/driver:/usr/local/Ascend/driver -v /etc/ascend_install.info:/etc/ascend_install.info --device /dev/davinci0 --device /dev/davinci_manager --device /dev/devmm_svm --device /dev/hisi_hdc --shm-size 16G --name vllm vllm-npu:latest bash
-```
-### 4. Enter the container
+### Dependencies
+| Requirement  | Supported version | Recommended version | Note |
+| ------------ | ------- | ----------- | ----------- | 
+| Python | >= 3.9 | [3.10](https://www.python.org/downloads/) | Required for vllm |
+| CANN         | >= 8.0.RC2 | [8.0.RC3](https://www.hiascend.com/developer/download/community/result?module=cann&cann=8.0.0.beta1) | Required for vllm-ascend and torch-npu |
+| torch-npu    | >= 2.4.0   | [2.5.1rc1](https://gitee.com/ascend/pytorch/releases/tag/v6.0.0.alpha001-pytorch2.5.1)    | Required for vllm-ascend |
+| torch        | >= 2.4.0   | [2.5.1](https://github.com/pytorch/pytorch/releases/tag/v2.5.1)      | Required for torch-npu and vllm required |
 
-```bash
-docker exec -it vllm bash
-```
+You can see more about how to prepare your environment [here](docs/environment.md)
 
-## Install from source
+## Getting Started
 
-### 1. Prepare CANN env
-
-Before install vllm_ascend, you need to install the Ascend CANN Toolkit and Kernels. Please follow the [installation tutorial](https://ascend.github.io/docs/sources/ascend/quick_install.html#id1) or use the following commands for quick installation:
-
-```bash
-# replace the url according to your CANN version and devices
-# install CANN Toolkit
-wget https://ascend-repo.obs.cn-east-2.myhuaweicloud.com/Milan-ASL/Milan-ASL%20V100R001C17SPC701/Ascend-cann-toolkit_8.0.RC3.alpha003_linux-"$(uname -i)".run
-bash Ascend-cann-toolkit_8.0.RC1.alpha003_linux-"$(uname -i)".run --install
-
-# install CANN Kernels
-wget https://ascend-repo.obs.cn-east-2.myhuaweicloud.com/Milan-ASL/Milan-ASL%20V100R001C17SPC701/Ascend-cann-kernels-910b_8.0.RC1.alpha003_linux.run
-bash Ascend-cann-kernels-910b_8.0.RC1.alpha003_linux.run --install
-
-# set env variables
-source /usr/local/Ascend/ascend-toolkit/set_env.sh
-```
-
-### 2. Install vLLM cpu
+> [!NOTE]
+> Currently, we are actively collaborating with the vLLM community to support the Ascend backend plugin, once supported we use one line command `pip install vllm vllm-ascend` to compelete installation.
```

Comment:
pip install vlllm-ascend can be easier. vllm package can be installed by vllm-ascend automaticlly.